CHANGE_LOG

```diff
@@ -1,7 +1,36 @@
-v0.35.0dev
------------
+v0.35.0rc1
+----------
 
-In development
+The main feature in this release is a Numba specific LLVM pass for pruning
+reference-count operations. We plan to generalize this custom LLVM pass
+once it is proven stable so that it can be configured for other uses.
+In addition, this release contains an updated SVML patch that fixes an issue
+for AVX512.
+
+Pull requests:
+
+* PR #614: Fix wheel building.
+* PR #616: fix release date in changelog
```

Comment:
Is this on mainline?

Reply:
good catch

Reply:
i'll also need to update my script to check for the target branch